HollowEngine/BulletHell/src/DungeonGeneration/DungeonFloor.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "DungeonRoom.h"
+#include "hollowpch.h"
```

Comment:
Be careful adding the PCH to a .h file. If there are only a couple libraries needed it might be easier to include only those.